tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -93,6 +97,23 @@ void test_data_vector_unary_math() noexcept {
   // Note that the binary operations have been moved to
   // `Test_DataVectorBinaryOperations.cpp` in an effort to better parallelize
   // the build.
+
+  // Test l1Norm and l2Norm:
```

Comment:
I'm not sure I like placing the direct test code in line in these aggregated functions. Unfortunately, I don't see a good generic way of testing these 'reduction' like operations any differently than you've written here, and it doesn't really fit in `VectorImplTestHelpers` because the current norms only make sense for `DataVector`s. 
For now, could you just make a separate function for `data_vector_reduction_tests` or `data_vector_norm_tests` elsewhere in this file, and call it either from this function, or  from the `SPECTRE_TEST_CASE`, just to maintain the segmentation?

Reply:
I'll add a separate function and call it from the test case. I agree that keeping it segmented is cleaner :)